src/parameterizations/vertical/MOM_diabatic_driver.F90

```diff
@@ -862,37 +827,31 @@ subroutine diabatic_ALE_legacy(u, v, h, tv, Hml, fluxes, visc, ADp, CDp, dt, Tim
 
     ! Augment the diffusivities and viscosity due to those diagnosed in energetic_PBL.
     do K=2,nz ; do j=js,je ; do i=is,ie
-      !### These expressions assume a Prandtl number of 1.
       if (CS%ePBL_is_additive) then
         Kd_add_here = Kd_ePBL(i,j,K)
-        visc%Kv_shear(i,j,K) = visc%Kv_shear(i,j,K) + Kd_ePBL(i,j,K)
+        visc%Kv_shear(i,j,K) = visc%Kv_shear(i,j,K) + CS%ePBL_Prandtl*Kd_ePBL(i,j,K)
```

Comment:
This is a bug fix that doesn't change answers for the M6E suite but does affect OM4.1 for which Pr=1.25 . We can forego a run-time parameter so long as no one else has a non-unity Pr.

Reply:
I introduced a new runtime parameter (PRANDTL_EPBL) that is used set ePBL_Prandtl with a default value of 1 so this can't possibly affect OM4.1.  We can consider using a global Prandtl number to set the defaults throughout the code, but this PR does not do this.